src/main/java/View/Web/Servlets/Privileged/UserSpecific/FlatInformationServlet.java

```diff
@@ -22,40 +23,49 @@
     public static final String SERVLET_URL =  "/action/projects/flats";
     public static final String TEMPLATE_NAME = "views/privileged/my_flat.html";
 
+    private static final String VARIABLE_FLAT = "flat";
+
+    private static final String REQUEST_PARAM_FLAT_ID = "fid";
+
     @Override
     protected void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
         // AUTHENTICATION
         super.doGet(request, response); // call always parent method first
-        if (checkPrivilege(request, response) == false) {
-            ServletHelper.SendReturnCode(response, HttpServletResponse.SC_UNAUTHORIZED);
-            return;
-        }
+//        if (checkPrivilege(request, response) == false) { // TODO uncomment back once UC finished
+//            ServletHelper.SendReturnCode(response, HttpServletResponse.SC_UNAUTHORIZED);
+//            return;
+//        }
 
         CustomLogs.Development("Sme vo doGet");
 
         // has to have some request for flat id
         int requestedFlatId;
         try {
-            requestedFlatId = Integer.parseInt(request.getParameter("fid"));
+            requestedFlatId = Integer.parseInt(request.getParameter(REQUEST_PARAM_FLAT_ID));
             CustomLogs.Development("V requeste prisiel flat id: " + requestedFlatId);
         } catch (NumberFormatException nfe) {
             CustomLogs.Error("Bad request or nothing came into server as ?fid=[number should be here]");
             ServletHelper.SendReturnCode(response, HttpServletResponse.SC_BAD_REQUEST);
             return;
         }
 
-        // Get user from session
-        User user = SessionUtil.getUser(request.getSession(false));
-        boolean hasHe = (new UC_FlatSummary(getDb())).temporaryCheckToSeeIfHasRightToSee(user.getUserID(), requestedFlatId);
-
         // TEMPLATE PREPARATION
         TemplateEngine engine = ContextUtil.getTemplateEngine(request.getServletContext());
         WebContext context = DcsWebContext.WebContextInitForDCS(request, response,
                 ConfigClass.HTML_VARIABLENAME_RUNNINGREMOTELY, trueIfRunningRemotely);
 
-
-        // Set variables on html
-        context.setVariable("RightToSeeFlat", hasHe);
+//        User user = SessionUtil.getUser(request.getSession(false));
+        UC_FlatSummary uc = new UC_FlatSummary(getDb());
+        if (uc.doesUserHaveRightToSeeProjectBelongingToFlat(1, requestedFlatId) == false) {
```

Comment:
^this mi zabilo dobre vela casu, hladal som chybu u mna. preco tu je 1ka natvrdo ved sme to mali uz implementovane ze to fungovalo zo session...

Reply:
prepac :( kym sa vyvyja som zakomentoval